src/utils/device.ts

```diff
@@ -65,11 +66,7 @@ const canAccessDeviceQuery = balenaApi.prepare<{ id: number }>({
 	id: { '@': 'id' },
 	url: `device(@id)/canAccess`,
 });
-export const canAccessDevice = (
-	device: DeviceInfo,
-	port: number,
-	auth?: Buffer,
-) =>
+const $canAccessDevice = (device: DeviceInfo, port: number, auth?: Buffer) =>
```

Comment:
If `auth` is a `Buffer` then I'm not sure how well that'll play with memoization

Reply:
@wrboyce ^ ?

Reply:
I use `normalize` to generate the cache header now, which takes this into account (`args[2] ?? 'guest'`).